linode/manager
# M3-5766: LKE UI refinements

## Description 📝

- Tighten up spacing in the Kubernetes Cluster details page
- Refactoring and code clean up
- Added compactY to Button component and renamed compact to compactX
- Updated Storybook Button to include compactY

## Preview 📷

Before
![image](https://user-images.githubusercontent.com/14323019/196509479-9f2b0037-6551-4ca1-acda-d0614c8e735a.png)

After
![image](https://user-images.githubusercontent.com/14323019/196509612-c7ac29c3-c754-44f7-8d36-8a2d28da5244.png)

## How to test 🧪
- Go to a Kubernetes cluster details page and check the changes
- Run Storybook locally (`yarn storybook`) and check the Button story
